code/modules/projectiles/gun_attachables.dm

```diff
@@ -1423,6 +1423,7 @@ Defined in conflicts.dm of the #defines folder.
 	if(bipod_deployed)
 		bipod_deployed = FALSE
 		to_chat(user, "<span class='notice'>You retract [src].</span>")
+		master_gun.aim_slowdown -= 1
```

Comment:
Atomize this change into a different PR so it doesn't get delayed with the review of the other ones.

Try to keep one PR per fix / feature whenever sensible.